doc/lua_api.txt

```diff
@@ -1666,6 +1666,7 @@ to games.
     * `2`: the node always gets the digging time 0.5 seconds (rail, sign)
     * `3`: the node always gets the digging time 0 seconds (torch)
 * `disable_jump`: Player (and possibly other things) cannot jump from node
+  or if their feet are in the node.
```

Comment:
```
Note: not supported for `new_move = false`
```
or similar. Document it so that people don't open "bug report" issues about this case.